packages/database/src/browser/MigrationManager.js

```diff
@@ -0,0 +1,141 @@
+import { PGlite } from '@electric-sql/pglite';
+import path from 'path';
+import { getConnectionConfig } from './getConnectionConfig';
+
+const DEFAULT_MIGRATIONS_PATH = path.resolve(__dirname, '/packages/database/src/core/migrations');
+
+export class MigrationManager {
+  client = null;
+  config = null;
+
+  constructor(config = {}) {
+    this.config = {
+      ...config,
+      migrationsPath: config.migrationsPath || DEFAULT_MIGRATIONS_PATH,
+      logger: config.logger || {
+        info: message => console.log(message),
+        error: message => console.error(message),
+      },
+    };
+    this.client = new PGlite(getConnectionConfig);
+  }
+
+  async initialize() {
+    await this.createMigrationsTable();
+  }
+
+  async close() {
+    await this.client.close();
+  }
+
+  async createMigrationsTable() {
+    await this.client.exec(`
+      CREATE TABLE IF NOT EXISTS migrations (
+        id SERIAL PRIMARY KEY,
+        version INTEGER NOT NULL,
+        name VARCHAR(255) NOT NULL,
+        executed_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
```

Comment:
I could check but does this match the format of db-migrate?

Reply:
Oh db-migrate has everything except for the version. I think it's redundant so I'll remove it

Reply:
Yep cool that's the one I was wondering about!